README.md

```diff
@@ -21,10 +21,25 @@ export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 export CLUSTER_ADDR=<cluster_name, eg: ravig211.devcluster.openshift.com>
 export KUBE_SSH_KEY_PATH=<path to ssh key>
 ```
-Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
+- Update the keypair mentioned at [sshKeyPair](https://github.com/openshift/windows-machine-config-operator/blob/42593c5d2eb798b572c58b5debafc4c392d1f967/test/e2e/wmco_test.go#L59) to match with what 
 is being used in KUBE_SSH_KEY_PATH. Please note that we're using libra as keypair
 for our CI purposes.
-
+- Make sure that `/payload` directory exists and has tree structure mentioned like this(Since this structure can change, please make sure this is upto date)
+```
+/payload/
+├── cni-plugins
+│   ├── flannel.exe
+│   ├── host-local.exe
+│   ├── win-bridge.exe
+│   └── win-overlay.exe
+├── hybrid-overlay.exe
+├── kube-node
+│   ├── kubelet.exe
+│   └── kube-proxy.exe
+├── powershell
+│   └── wget-ignore-cert.ps1
+└── wmcb.exe
+```
 Once the above variables are set:
```

Comment:
`Once the above variables are set and the /payload directory has been populated, run the following script:`